src/consensus/vote.cpp

```diff
@@ -230,41 +226,92 @@ uint64_t VoteManager::getUnverifiedVotesSize() const {
   return size;
 }
 
+std::vector<Vote> VoteManager::getVerifiedVotes() {
+  std::vector<Vote> votes;
```

Comment:
isn't this same thing, that @JakubFornadel pointed out where we are doing a lot of allocations? Why we are not using shared_ptr also here?

Reply:
Fixed, thanks